frontend/src/pages/Q&A/AnswerModel/AnswerModel.scss

```diff
@@ -123,7 +123,14 @@
     margin: 5px;
     cursor: pointer;
   }
-  
+.most-relevant-label{
```

Comment:
can we make this inline with the username? Also, lets remove the time from ui, just date will should be enough